impl/events.go

```diff
@@ -244,17 +250,20 @@ func (m *manager) OnChannelCompleted(chid datatransfer.ChannelID, completeErr er
 	if completeErr == nil {
 		// If the channel was initiated by the other peer
 		if chid.Initiator != m.peerID {
+			log.Infow("received OnChannelCompleted, will send completion message to initiator", "chid", chid)
 			msg, err := m.completeMessage(chid)
 			if err != nil {
-				return nil
+				return err
```

Comment:
Is this changed on purpose? I see everything else is log lines, apart from this.

Reply:
This needs an explanation, as it introduces a change in behaviour.

Reply:
Hey,

The error returned from here is only logged and we take no action based on it. If there's an error while creating the complete message, we do want to know about it.